go.mod

```
@@ -16,17 +16,17 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.4
 	gonum.org/v1/gonum v0.12.0
-	k8s.io/api v0.28.4
-	k8s.io/apimachinery v0.28.4
-	k8s.io/apiserver v0.28.4
-	k8s.io/client-go v0.28.4
-	k8s.io/code-generator v0.28.4
-	k8s.io/component-base v0.28.4
-	k8s.io/component-helpers v0.28.4
+	k8s.io/api v0.28.8
+	k8s.io/apimachinery v0.28.8
+	k8s.io/apiserver v0.28.8
+	k8s.io/client-go v0.28.8
+	k8s.io/code-generator v0.28.8
+	k8s.io/component-base v0.28.8
+	k8s.io/component-helpers v0.28.8
 	k8s.io/klog/hack/tools v0.0.0-20210917071902-331d2323a192
 	k8s.io/klog/v2 v2.100.1
-	k8s.io/kube-scheduler v0.28.4
-	k8s.io/kubernetes v1.28.4
+	k8s.io/kube-scheduler v0.28.8
+	k8s.io/kubernetes v1.28.8
 	k8s.io/utils v0.0.0-20230406110748-d93618cff8a2
 	sigs.k8s.io/controller-runtime v0.15.0
```

```suggestion
	sigs.k8s.io/controller-runtime v0.16.5
```

@Huang-Wei Could we upgrade the controller-runtime version as well? Because the v0.16.5 is corresponding to k8s v1.28: https://github.com/kubernetes-sigs/controller-runtime/blob/dca5e8b2b00a1ab10a617f6c4a6fd0d40f9a974e/go.mod#L23-L27

---

sure, updated.

---

Thank you!